proto/responses.proto

```diff
@@ -62,6 +62,20 @@ message SyncStateResponse {
     repeated NullifierUpdate nullifiers = 8;
 }
 
+message SyncNoteResponse {
+    // Number of the latest block in the chain
+    fixed32 chain_tip = 1;
+
+    // Block header of the block with the first note matching the specified criteria
+    block_header.BlockHeader block_header = 2;
+
+    // Data needed to update the partial MMR from `request.block_num + 1` to `response.block_header.block_num`
+    mmr.MmrDelta mmr_delta = 3;
```

Comment:
Based on recent discussions, I think this should be just the `MmrProof` for the included block header against the latest chain MMR on the node.